### PR TITLE
More concise output. More tankaesque, mk2

### PR DESCRIPTION
**Pull request by malcolmholmes**

### CODE REVIEW

File: pkg/dash/workflow.go [hunk 1]
```diff
@@ -101,11 +110,26 @@ func Apply(config Config, jsonnetFile string, targets *[]string) error {
...
+
+		if boardJSON == existingBoardJSON {
+			fmt.Println(name, yellow("unchanged"))
+		} else {
+			fmt.Println(name, green("updated"))
+
+			err = postDashboard(config, board)
+			if err != nil {
+				return err
+			}
```
[the remark this answers is not in the document]
Comment (malcolmholmes) [reply pkg/dash/workflow.go:132]: The intent is to give the user clear feedback. "Unchanged" vs "updated" allows the user to see that the change they thought they were applying matched what happened.

File: pkg/dash/workflow.go [hunk 2]
```diff
@@ -27,7 +34,9 @@ func List(jsonnetFile string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println(strings.Join(keys, "\n"))
+	for _, key := range keys {
+		fmt.Println(key, yellow("found"))
+	}
```
[the remark this answers is not in the document]
Comment (malcolmholmes) [reply pkg/dash/workflow.go]: The key is to report that this is found in jsonnet not in Grafana. I've removed 'found' which as you say doesn't add much, and added a 'Found in jsonnet' message before the list.

File: pkg/dash/workflow.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (malcolmholmes) [reply pkg/dash/workflow.go]: Yes, I broke apply. I've fixed it, and tidied it up more now. So it now says, `foo.jsonnet added` when a dashboard does not already exist.

[the remark this answers is not in the document]
Comment (trotttrotttrott) [reply pkg/dash/workflow.go]: After seeing it working, I agree it's nice to have 👍

**Pull request merged**